response.go

```diff
@@ -55,8 +74,14 @@ func PreloadableResponse(resp *http.Response) (header *string, issues Issues) {
 // `header` is `nil`.
 // To interpret `issues`, see the list of conventions in the
 // documentation for Issues.
-func RemovableResponse(resp *http.Response) (header *string, issues Issues) {
-	return checkResponse(resp, RemovableHeaderString)
+func RemovableResponse(resp *http.Response, policy preloadlist.PolicyType) (header *string, issues Issues) {
+	return checkResponse(resp, RemovableHeaderStringWrapper, policy)
+}
```

Comment:
What does RemovebleResponse with a policy mean? I don't think this change should exist.